apps/fortuna/src/keeper/fee.rs

```diff
@@ -54,16 +109,65 @@ pub async fn withdraw_fees_if_necessary(
         .await
         .map_err(|e| anyhow!("Error while getting provider info. error: {:?}", e))?;
 
-    if provider_info.fee_manager != wallet.address() {
-        return Err(anyhow!("Fee manager for provider {:?} is not the keeper wallet. Fee manager: {:?} Keeper: {:?}", provider, provider_info.fee_manager, wallet.address()));
-    }
-
     let fees = provider_info.accrued_fees_in_wei;
 
     if keeper_balance < min_balance && U256::from(fees) > min_balance {
         tracing::info!("Claiming accrued fees...");
+
         let contract_call = contract.withdraw_as_fee_manager(provider_address, fees);
         send_and_confirm(contract_call).await?;
+
+        let fee_manager_wallet = LocalWallet::from_str(&fee_manager_private_key)
+            .map_err(|e| anyhow!("Invalid fee manager private key: {:?}", e))?;
+        let fee_manager_address = fee_manager_wallet.address();
+        let keeper_address = wallet.address();
+
+        if fee_manager_address != keeper_address {
+            tracing::info!(
+                "Transferring withdrawn fees from fee manager {:?} to keeper {:?}",
+                fee_manager_address,
+                keeper_address
+            );
+
+            let transfer_amount = U256::from(fees);
```

Comment:
(aside) there's an issue here where the fee manager wallet could run out of funds to do the transfer. probably need to set a minimum balance on the fee manager wallet and then transfer something a bit less than that.